### PR TITLE
Fixed licenses and notice file

### PR DESCRIPTION
**Pull request by merlimat**
### Motivation

Some new dependencies were missing from LICENSE and NOTICE files.

### CODE REVIEW

File: all/src/assemble/LICENSE.bin.txt [hunk 1]
```diff
@@ -338,16 +338,18 @@ The Apache Software License, Version 2.0
 
 BSD 3-clause "New" or "Revised" License
  * EA Agent Loader -- com.ea.agentloader-*.jar -- licenses/LICENSE-EA-Agent-Loader.txt
+ * Google auth library - com.google.auth-google-auth-library-*.jar -- licenses/LICENSE-google-auth-library.txt
```
Comment (rdhabalia) [inline all/src/assemble/LICENSE.bin.txt]: does pulsar depend on specific google-auth binary?

File: all/src/assemble/NOTICE.bin.txt [hunk 2]
```diff
@@ -94,13 +94,82 @@ Copyright 2004 Jason Paul Kitchen
...
 Copyright 2012-2015 The Prometheus Authors
 
+This product includes software developed at
+Boxever Ltd. (http://www.boxever.com/).
+
+This product includes software developed at
+SoundCloud Ltd. (http://soundcloud.com/).
+
+This product includes software developed as part of the
+Ocelli project by Netflix Inc. (https://github.com/Netflix/ocelli/).
```
Comment (rdhabalia) [inline all/src/assemble/NOTICE.bin.txt]: just wanted to check which binary belong to this one,?

Comment (rdhabalia) [reply all/src/assemble/NOTICE.bin.txt]: I see.. so, we have to mention license/notice for transitive dependencies as well?

Comment (merlimat) [reply all/src/assemble/NOTICE.bin.txt]: That we are always doing. Here is reporting in our NOTICE file what is in the NOTICE file from our dependencies (both first level and transitives). In particular this refers to libraries/code that is bundled inside a particular dependency.

File: all/src/assemble/NOTICE.bin.txt [hunk 3]
```diff
@@ -94,13 +94,82 @@ Copyright 2004 Jason Paul Kitchen
...
+    * http://tomcat.apache.org/native-doc/
+* https://svn.apache.org/repos/asf/tomcat/native/
+
+----------------------------------------------------------------------------------------------------
+
+Jetty Web Container
+Copyright 1995-2016 Mort Bay Consulting Pty Ltd.
+
+The Jetty Web Container is Copyright Mort Bay Consulting Pty Ltd
+unless otherwise noted.
```
Comment (ivankelly) [inline all/src/assemble/NOTICE.bin.txt]: Missing: https://github.com/eclipse/jetty.project/blob/d4e96559ab42b9a36db53885bec93955255d0beb/NOTICE.txt#L113

It's part of jetty-util

**Comment by ivankelly** [comment]
#shipit

**Comment by merlimat** [comment]
retest this please

**Pull request merged**